projects/plugins/boost/app/assets/src/js/pages/settings/sections/Support.svelte

```diff
@@ -0,0 +1,41 @@
+<script>
+	/**
+	 * Internal dependencies
+	 */
+	import ReactComponent from '../../../elements/ReactComponent.svelte';
+	import { isPaidPlan, openPaidSupport } from '../../../utils/paid-plan';
+	/**
+	 * WordPress dependencies
+	 */
+	import { __ } from '@wordpress/i18n';
+	import { Button } from '@wordpress/components';
+</script>
+
+{#if isPaidPlan}
+	<div class="jb-section">
+		<div class="jb-container--narrow">
+			<div class="jb-support">
+				<div class="jb-support__content">
+					<h3 class="jb-support__title">
+						{__( "We're here to help", 'jetpack-boost' )}
+					</h3>
+					<p class="jb-support__description">
+						{__(
+							'Your paid plan gives you access to prioritized Jetpack Boost support',
+							'jetpack-boost'
+						)}
+					</p>
+				</div>
+				<div class="jb-support__cta">
+					<ReactComponent
+						this={Button}
+						label={__( 'Continue', 'jetpack-boost' )}
+						onClick={openPaidSupport}
+						className="jb-button jb-button--outline"
+						children={__( 'Contact Us', 'jetpack-boost' )}
+					/>
```

Comment:
Why use the React Button component, when it's just a normal HTML button with styles?
```
<button class="components-button jb-button jb-button--outline">
  Contact Us
</button>
```